ansible/roles/lp-synctool-deploy/templates/application.conf.j2

```diff
@@ -120,4 +120,7 @@ csp.migration.request.topic="{{ csp_migration_topic_name }}"
 csp.migration.batch.size={{ csp_migration_batch_size }}
 is_replace_string={{ sync_tool_is_replace_string | default('false') }}
 replace_src_string= "{{ sync_tool_replace_src_string | default('') }}"
-replace_dest_string="{{ sync_tool_replace_dest_string | default('') }}"
\ No newline at end of file
+replace_dest_string="{{ sync_tool_replace_dest_string | default('') }}"
+
+replace_src_string_DIAL_store= "{{ sync_tool_replace_src_string_DIAL_store | default('') }}"
+replace_dest_string_DIAL_store="{{ sync_tool_replace_dest_string_DIAL_store | default('') }}"
```

Comment:
Why are we replacing the paths. Please give an example to understand...

Reply:
Added